packages/block-library/src/post-content/index.php

```diff
@@ -44,7 +44,7 @@ function render_block_core_post_content( $attributes, $content, $block ) {
 
 	$seen_ids[ $post_id ] = true;
 
-	if ( ! in_the_loop() ) {
+	if ( ! in_the_loop() && have_posts() ) {
```

Comment:
This change is about this notice:
>PHP Notice:  Trying to access array offset on value of type null in /var/www/html/wp-includes/class-wp-query.php on line 3300

@aristath would like some input if you think this affects the results we want, since you had introduced this. In my testing though it seems to work as before.

Reply:
Makes sense to use `have_posts()` :+1: